worlds/shivers/__init__.py

```diff
@@ -78,17 +83,35 @@ def create_items(self) -> None:
         #Add items to item pool
         itempool = []
         for name, data in item_table.items():
-            if data.type in {"pot", "key", "ability", "filler2"}:
+            if data.type in {"key", "ability", "filler2"}:
                 itempool.append(self.create_item(name))
 
+        # Pot pieces/Completed/Mixed:
+        for i in range(10):
+            if self.options.full_pots == 0:
```

Comment:
Preferred is to compare against a string: `if self.options.full_pots == "pieces"`
or even better: `if self.options.full_pots == FullPots.option_pieces`. That requires an import, but it's the safest and most explicit

Reply:
There are more instances of this, I won't mention all of them individually, please have a look yourself

Reply:
Fixed several (all?)